src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -601,6 +601,10 @@ public static class ChartFontOptions
       public static class ImporterConstants
       {
          public static readonly string[] NAMING_PATTERN_SEPARATORS = { ".", ",", "-", "_" };
+
+         public const string DATA_SETS_TO_BE_DELETED = "toDeleteDataSets";
```

Comment:
Those are constants that are used for the Cache that ReloadFromConfiguration is returning. If we switch back to a triple they will no longer be necessary, otherwise I guess they are in the correct place, right?

Reply:
removed, since the return value type has changed